pkg/kubelet/cm/cpumanager/state/checkpoint.go

```diff
@@ -110,8 +110,9 @@ func (cp *CPUManagerCheckpointV1) VerifyChecksum() error {
 
 	hash := fnv.New32a()
 	fmt.Fprintf(hash, "%v", object)
-	if cp.Checksum != checksum.Checksum(hash.Sum32()) {
-		return errors.ErrCorruptCheckpoint
+	actualCS := checksum.Checksum(hash.Sum32())
+	if cp.Checksum != actualCS {
+		return &errors.CorruptCheckpointError{ActualCS: uint64(actualCS), ExpectedCS: uint64(cp.Checksum)}
```

Comment:
Can you pull these (arguably) unrelated changes to the checkpoint error into their own commit?

Reply:
Ack

Reply:
Also moved into separate PR: https://github.com/kubernetes/kubernetes/pull/126156